website/docs/r/endpoints_service.html.markdown

```diff
@@ -35,7 +35,7 @@ The following arguments are supported:
 * `service_name`: (Required) The name of the service.  Usually of the form `$apiname.endpoints.$projectid.cloud.goog`.
 * `openapi_config`: (Optional) The full text of the OpenAPI YAML configuration as described [here](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).  Either this, or *both* of `grpc_config` and `protoc_output` must be specified.
 * `grpc_config`: (Optional) The full text of the Service Config YAML file (Example located [here](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/endpoints/bookstore-grpc/api_config.yaml)).  If provided, must also provide `protoc_output`.  `open_api` config must *not* be provided.
-* `protoc_output`: (Optional) The full contents of the Service Descriptor File generated by protoc.  This should be a compiled .pb file.
+* `protoc_output_base64`: (Optional) The full contents of the Service Descriptor File generated by protoc.  This should be a compiled .pb file, base64-encoded.
```

Comment:
we usually still keep the old field, but just mark it deprecated.

Reply:
Done.